test/test-opensuse.sh

```diff
@@ -8,7 +8,7 @@ if [ ! -f ${PKG_FILE} ]; then
     exit 0
 fi
 
-zypper --non-interactive install ${PKG_FILE}
+zypper --non-interactive --no-gpg-checks install ${PKG_FILE}
```

Comment:
Oh, this brings up an interesting question -- should we start signing our installers? Jenkins is already configured with the secrets we need for this.

Reply:
Yeah, I think that'd be great. We still have this issue open: https://github.com/rstudio/r-builds/issues/7